volto/src/addons/volto-govuk-theme/src/customizations/volto/components/theme/Search/Search.jsx

```diff
@@ -231,66 +251,25 @@ class Search extends Component {
                       defaultMessage="results"
                     />
                   </div>
-                  {/* <Header
+                  { <Header
                     style={{
                       marginTop: 0,
                     }}
                   >
                     <Header.Content className="header-content">
-                      <div className="govuk-body-xl sort-by">
-                        <FormattedMessage
-                          id="Sort By:"
-                          defaultMessage="Sort by:"
-                        />
+                      <div class="govuk-form-group">
+                        <label class="text-aling: left" style={{marginRight: "20px", textAlign: "left"}} for="subject">
+                          Sort By
+                        </label>
+                        <select class="govuk-select" id="subject" name="subject" aria-describedby="subject-hint" onChange={this.onSortChange} value={this.state.active}>
+                          <option value="Choose">Choose</option>
+                          <option value="relevance">relevance</option>
```

Comment:
Capitalise relevance to make it uniform with the other options.
```suggestion
                          <option value="relevance">Relevance</option>
```